layers/+misc/nav-flash/funcs.el

```diff
@@ -0,0 +1,34 @@
+;;; funcs -*- lexical-binding: t; -*-
```

Comment:
Need to add header
https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.org#file-header